.gitignore

```diff
@@ -1,14 +1,6 @@
-
-dist
 bazel-out
 node_modules
 templates/
-bin/*
```

Comment:
I'm hesitant to say that the compiled bundle.js should be included in these pushes since that will change every single time and show a larger than necessary commit and bundle.js will most definitely encounter conflicts when any change is made that will be hard to sort out with commits.

Reply:
Main reason to ship the compiled js is so that anyone can then download it and use in chrome without  installing npm and building it them self. But i see your point. Alternative is to make a dist folder and put everything there that is ready to use